### PR TITLE
MOB-3843 Align gRPC version between go and java

### PR DESCRIPTION
**Pull request by codingrhythm**
## Summary

The Java code gen is not using the same protobuf&grpc version as Golang. This PR updates the java code gen plugin that aliges with 1.27.0 of gRPC

## Checklist

- [ ] Is this PR a reasonable size?

---

**Code Review Guidelines** for Reviewers

- Try to review in a timely manner. Opinions/nitpicks should not be blockers. Pair on a call for non-trivial feedback.
- Overall design and approach should follow established patterns. Don't try to make the PR perfect.
- Try to identify edge cases, race conditions, over-engineering, lack of test coverage and complexity.
- If you don't feel qualified to review the code, pass it on to someone who is.

### CODE REVIEW

File: protoc-java/Dockerfile [hunk 1]
```diff
@@ -1,26 +1,12 @@
 FROM ubuntu:16.04 as javalite_builder
 RUN apt-get update && \
     apt-get install -y bash patch curl patchelf
-ENV PROTOC_GEN_JAVALITE_VERSION=3.0.0
```
Comment (codingrhythm) [inline protoc-java/Dockerfile:4]: java plugin is part of the protoc now, so we no longer need to install separately

**Review by MickStanciu — APPROVED** [review]
